www/include/configuration/configObject/contact/formContact.php

```diff
@@ -784,9 +786,14 @@ function myReplace()
 
 if ($form->validate() && $from_list_menu == false) {
     $cctObj = $form->getElement('contact_id');
+    if (!$centreon->user->admin && $contact_id) {
```

Comment:
you should have replaced all $contact_id with $contactId to respect the camelCase rule :'(